README.md

```diff
@@ -126,14 +126,14 @@ The corresponding Python component API is auto-discovered from the React compone
 
 ```javascript
 /** Used to auto-generate the Python component and docstrings */
-export default class NeptuneComponent extends Component {
-  render() {
-    const {text} = this.props;
-
-    return <div>
-             <p>{text}</p>
-           </div>;
-  }
+function NeptuneComponent(props) {
```

Comment:
🐼 

As we decided to enforce a "fat-arrow" pattern for React functional components, maybe relevant to precise such a pattern onto the library `README`?

Reply:
Done! 👍